contracts/token/ERC20/extensions/ERC20Votes.sol

```diff
@@ -24,7 +26,7 @@ import "../../../utils/cryptography/ECDSA.sol";
  *
  * _Available since v4.2._
  */
-abstract contract ERC20Votes is IVotes, ERC20Permit {
+abstract contract ERC20Votes is ERC20, IVotes, Nonces, EIP712 {
```

Comment:
We need this to use `Votes`. Currently it defines its own vote tracking and delegation logic, it should reuse that instead. See `ERC721Votes` for reference.